chart/templates/_helpers.yaml

```diff
@@ -309,6 +309,10 @@ If release name contains chart name it will be used as a full name.
   {{- if .Values.dags.gitSync.extraVolumeMounts }}
     {{- tpl (toYaml .Values.dags.gitSync.extraVolumeMounts) . | nindent 2 }}
   {{- end }}
+  {{- $containerLifecycleHooksGitSync := or .Values.dags.gitSync.containerLifecycleHooks .Values.containerLifecycleHooks }}
```

Comment:
> The intent of the global one is for Airflow containers. Since this isn't an Airflow, we shouldn't use the global here.
> 
> Can you also update the [comment here in values](https://github.com/apache/airflow/blob/e8a8208477a4dd91d4b89b3ca5c964a3ea69d2ab/chart/values.yaml#L60)? It's probably easier to say "for airflow" vs listing the exceptions.

`logGroomerSidecar`, `kerberosSidecar` container also has global containerLifecycleHook. is it 'Airflow container's?
https://github.com/getChan/airflow/blob/f81abd6145d20305148e0a91e8a6ff5026927b70/chart/templates/workers/worker-deployment.yaml#L39-L40

Reply:
Those are definitely a bit more blurry, since they use the Airflow image :)

Log groomer, no. It isn't running "airflow". If you look, it also doesn't include the global env vars either, for example.

Kerberos, yes. That is an Airflow command that is ultimately being run.

Reply:
applied your review